zebra-state/src/lib.rs

```diff
@@ -85,6 +91,7 @@ impl Config {
             .path(path)
             .cache_capacity(self.memory_cache_bytes)
             .mode(sled::Mode::LowSpace)
+            .temporary(self.ephemeral)
```

Comment:
I chose the later for now.

Reply:
> > Otherwise, we should stop calling `Config::path` when `ephemeral` is true.
>
> I chose the latter for now. If anyone asks for supporting temporary paths then we can change the behaviour.

I can't see this change anywhere. Are you planning on making it in a future commit?

Reply:
I actuallly wrote that change but rolled it back due to tes failure. Let me add it back.

Reply:
Looks good now!